api/src/main/java/com/datatorrent/api/Operator.java

```diff
@@ -99,6 +99,24 @@ public String toString()
   }
 
   /**
+   * DelayOperator is an operator that increments the outgoing streaming window
+   * id by one and allows loops in the "DAG". It is meant for iterative algorithms
```

Comment:
Yes, but not at the platform level.  An implementation of DelayOperator can implement offset that is greater than one.